samples/batch/transcription-enabled-storage/Setup/ArmTemplate.json

```diff
@@ -190,7 +177,7 @@
             }
         },
         "PunctuationMode": {
-            "defaultValue": "DictatedAndAutomatic",
+            "defaultValue": "None",
```

Comment:
Is by intention to set punctuation mode to None? The default mode for transcription is DictatedAndAutomatic.

Reply:
okay, I'll set the default to DictatedAndAutomatic. Is the default for ProfanityFIlterMode also != None?